src/plugins/notebook/utils/notebook-entries.js

```diff
@@ -159,7 +159,7 @@ export async function addNotebookEntry(
     return;
   }
 
-  const date = Date.now();
+  const date = openmct.time.now();
```

Comment:
Can you confirm that notebook embeds are also using the clock time and not Date.now()?
Like when saving a snapshot?